src/pages/Troubleshooting.vue

```diff
@@ -1,36 +1,109 @@
 <template>
-  <ul class="troubleshooting">
-    <li>
-      <label>
-        <button
-          id="btnTroubleShootingFactoryReset"
-          :disabled="!canFactoryReset"
-          class="btn role-secondary"
-          @click="factoryReset"
-        >
-          Factory Reset
-        </button>
-        Factory reset will remove all Rancher Desktop configuration.
-      </label>
-    </li>
-    <li>
-      <button class="btn role-secondary" @click="showLogs">
-        Show Logs
-      </button>
-    </li>
-  </ul>
+  <section class="dashboard">
+    <header>
+      <div class="title">
+        <h1>{{ t('troubleshooting.title') }}</h1>
+      </div>
+      <hr>
+      <span class="description">
+        {{ t('troubleshooting.description') }}
+      </span>
+    </header>
+    <section class="troubleshooting">
+      <section class="kubernetes">
+        <h2>{{ t('troubleshooting.kubernetes.title') }}</h2>
+        <troubleshooting-line-item>
+          <template #title>
+            {{ t('troubleshooting.kubernetes.resetKubernetes.title') }}
+          </template>
+          <template #description>
+            {{ t('troubleshooting.kubernetes.resetKubernetes.description') }}
+          </template>
+          <button
+            type="button"
+            class="btn btn-xs role-secondary"
+            :disabled="cannotReset"
+            @click="reset"
+          >
+            {{ t('troubleshooting.kubernetes.resetKubernetes.buttonText') }}
+          </button>
```

Comment:
Resetting Kubernetes isn't a troubleshooting task. It's a more common task than that.

For example, if you're testing the installation of an application you'll want to quickly restore to an empty environment. Hence, you use the reset button. There are many other times, besides troubleshooting, that you would want to reset Kubernetes.

The same applies to container images.

I found both of these while interviewing people for the feature.

I'm not sure the troubleshooting page makes this very discoverable. Can you put those back on the Kubernetes settings page? Or, is there a good reason to have them under troubleshooting.

Reply:
I think "Reset Kubernetes" should be on the Kubernetes tab, but "Factory Reset" should be on a VM setting page.

This will become more important once we support `k3d` via `moby`, when you really don't want to wipe away your k3d clusters when you just wanted to wipe your k3s environment.

Reply:
With this change these actions would be represented in both Troubleshooting and Kubernetes Settings; it will be simple enough to drop these actions entirely from troubleshooting. In hindsight, it probably makes most sense to keep these actions in a single location.

Reply:
For now, I think we'll keep this as a simple redesign of our existing troubleshooting page. We can make further refinements when we have a better idea of how we'd like to support and interact with k3d.